src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -642,26 +632,16 @@ export default class AirportModel {
         return this._runwayCollection.getRunwayRelationshipForRunwayNames(primaryRunwayName, comparatorRunwayName);
     }
 
-    // TODO: Implement changing winds, then bring this method back to life
     /**
      * @for AirportModel
      * @method updateRunway
      */
     updateRunway() {
-        // const bestRunwayForWind = this._runwayCollection.findBestRunwayForWind(this.getWind);
-        //
-        // this.setArrivalRunway(bestRunwayForWind);
-        // this.setDepartureRunway(bestRunwayForWind);
-    }
+        const bestRunwayForWind = this._runwayCollection.findBestRunwayForWind(this.getWind);
```

Comment:
you could either call the function as a parameter here or set it to a const and pass it along:
```javascript
// option one
const bestRunwayForWind = this._runwayCollection.findBestRunwayForWind(getWind());

// option two (preferred)
const currentWind = getWind();
const bestRunwayForWind = this._runwayCollection.findBestRunwayForWind(currentWind);
```